rpcd/playbooks/elasticsearch.yml

```diff
@@ -54,6 +56,27 @@
       tags:
         - elasticsearch-pre-install
   roles:
-    - "elasticsearch"
+    - role: "ansible-elasticsearch"
+      es_instance_name: "openstack"
```

Comment:
So the vars in group vars have it listening on localhost, which won't work in a real ELK deployment.  Can I check on `{{ container_address }}` here during the artifact job?